studio/localStores/storageExplorer/StorageExplorerStore.js

```diff
@@ -291,6 +294,8 @@ class StorageExplorerStore {
       return this.removeTempRows(columnIndex)
     }
 
+    this.updateFolderAfterEdit(formattedName, columnIndex)
```

Comment:
Shifted it up a level as there's an awkward pause from hitting "Enter" until the folder is ready as we're uploading the placeholder file.

We can just let the user access the folder immediately without waiting for the placeholder file to upload finish